src/stores/contactmoment/index.ts

```diff
@@ -11,35 +11,66 @@ import { defineStore } from "pinia";
 import { createSession, type Session } from "../switchable-store";
 export * from "./types";
 import type { ContactVerzoekVragenSet } from "@/features/contactverzoek/formulier/types";
-import type { Afdeling } from "@/composables/afdelingen";
 export type ContactmomentZaak = { zaak: ZaakDetails; shouldStore: boolean };
 
-export interface ContactverzoekAfdeling {
+export interface Afdeling {
+  id: string;
+  identificatie: string;
+  naam: string;
+}
+
+export interface Groep {
+  id: string;
+  afdelingId: string;
+  identificatie: string;
+  naam: string;
+}
+
+export interface MederwerkerGroepAfdeling {
   id: string;
   identificatie: string;
   naam: string;
 }
 
 export interface MedewerkerAfdelingen {
   afdelingId: string;
+  id: string;
+  identificatie: string;
+  afdelingnaam: string;
+}
+
+export interface MedewerkerGroepen {
+  afdelingId: string;
+  id: string;
+  identificatie: string;
+  groepsnaam: string;
+}
+
+export interface ContactVerzoekMedewerker
+{
+  user: string;
+  identificatie?: string;
+  voornaam?: string;
+  voorvoegselAchternaam?: string;
+  achternaam?: string;
+  afdelingen: MedewerkerAfdelingen[];
+  groepen: MedewerkerGroepen[];
 }
 
 export type ContactmomentContactVerzoek = {
   url?: string;
   isMedewerker?: true;
-  medewerker?: {
-    user: string;
-    identificatie?: string;
-    voornaam?: string;
-    voorvoegselAchternaam?: string;
-    achternaam?: string;
-    afdelingen: MedewerkerAfdelingen[];
-  };
-  afdeling?: ContactverzoekAfdeling;
-  groep?: {
-    identificatie: string;
-    naam: string;
-  };
+  selectedOption?: 'afdeling' | 'medewerker' | 'groep'; 
+  afdeling?: Afdeling;
```

Comment:
kan je de aanpassing aan het model toelichten? een contact moment had eerst een medewerker en/of een afdeling en/of een groep. dat is toch nog steeds het geval?

Reply:
voor nu negeren nemen we mee naar een volgende refactoring story